fsharp-backend/src/BwdServer/Server.fs

```diff
@@ -386,6 +386,50 @@ let runDarkHandler (ctx : HttpContext) : Task<HttpContext> =
 
           return! unmatchedRouteResponse ctx requestPath route
 
+      | [ { spec = PT.Handler.HTTPBytes (route = route); tlid = tlid } as handler ] ->
+        Telemetry.addTags [ "handler.route", route; "handler.tlid", tlid ]
+
+        let routeVars = Routing.routeInputVars route requestPath
+
+        let! reqBody = getBody ctx
+        let reqHeaders = getHeaders ctx
+        let reqQuery = getQuery ctx
+
+        match routeVars with
+        | Some routeVars ->
+          Telemetry.addTag "handler.routeVars" routeVars
+
+          // Do request
+          use _ = Telemetry.child "executeHandler" []
+
+          let request =
+            HttpMiddlewareV1.Request.fromRequest url reqHeaders reqQuery reqBody
+          let inputVars = routeVars |> Map |> Map.add "request" request
+          let! (result, _) =
+            RealExe.executeHandler
+              canvas.meta
+              (PT2RT.Handler.toRT handler)
+              (Canvas.toProgram canvas)
+              traceID
+              inputVars
+              (RealExe.InitialExecution(desc, request))
+
+          let result = HttpMiddlewareV1.Response.toHttpResponse result
+          let result = HttpMiddlewareV1.Cors.addCorsHeaders reqHeaders result
```

Comment:
Why is cors stuff still needed?

Reply:
I thought there might be some localhost-testing reason to include it, but that was fuzzy. I'll remove it and see if anything breaks, thanks!